cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -197,10 +195,12 @@ def render_and_launch_ingress(kube_dns):
 def scale_ingress_controller():
     ''' Scale the number of ingress controller replicas to match the number of
     nodes. '''
-    output = kubectl('get', 'nodes', '-o', 'name')
-    count = len(output.splitlines())
-    if data_changed('ingress-controller-replicas', count):
```

Comment:
In testing, running `kubectl scale` where the scale doesn't change appears to be a noop, so removing this check.